### ISSUE TITLE
这个怎么卖空啊

### ISSUE THREAD

**Issue by wx2480** [kernc/backtesting.py#65]
### Steps to Reproduce

1.
2.
3.

### Additional info

- Backtesting version:

**Comment by kernc** [comment]
Please add more information to your issue report, preferably in English. :grimacing:

**Comment by wx2480** [comment]
How do you short a stock，and if i want to use a fifth of the money to buy stocks, how can i achieve it?

**Comment by kernc** [comment]
With #47 merged and 0.2.0 released, you can short using one fifth of equity with:
```py
class MyStrategy(Strategy):
   ...
   def next(self):
       ...
       self.sell(size=.5)
```
